WalletWasabi/Tor/Http/ClearnetHttpClient.cs

```diff
@@ -15,11 +15,15 @@ public class ClearnetHttpClient : IRelativeHttpClient
 	{
 		static ClearnetHttpClient()
 		{
-			HttpClient = new HttpClient(new HttpClientHandler()
+			var socketHandler = new SocketsHttpHandler()
 			{
-				AutomaticDecompression = DecompressionMethods.GZip,
-				SslProtocols = IHttpClient.SupportedSslProtocols
-			});
+				AutomaticDecompression = DecompressionMethods.All,
+				PooledConnectionLifetime = TimeSpan.FromMinutes(5)
```

Comment:
I looked around on the internet. I saw settings from 1 to 10 minutes in different projects. We don't have that much HTTP load, also we are far better than before with reusing connections - so I would go with 1 minute. 
@lontivero what do you think about this?

Reply:
I think `ClearnetHttpClient` is used only for testing or in really exceptional circumstances (I had to used it once in an airport, once in three years) so, it is not something critical. We have never had any problem with the number of sockets available neither, or at least not in the client side.

IMO this could be an improvement and five minutes is a reasonable lifetime.  OTOH the I think the `AutomaticDecompression = DecompressionMethods.All` is not okay because the client explicitly indicates the accepted encoding (gzip) in the request and then it should only accept gzip. Also, take into account that the coordinator is our enemy, we cannot trust on it. We should be always careful about what we accept from it.

Reply:
Is HttpClient in Wasabi Walelt used only to communicate with backend server or are there any other uses?

Reply:
I should check but I think it is only to communicate with the coordinator.

Reply:
1 minute is quite aggressive setting. It means basically that you expect that your DNS records can change any time. If everyone on Internet had that setting, DNS servers would be overloaded, I guess. I think we are fine with 5 minutes.

Reply:
Ok, this is fine like this then GJ.